platform/darwin/src/MGLLocationManager.m

```diff
@@ -38,7 +38,15 @@ - (CLLocationAccuracy)desiredAccuracy {
 }
 
 - (CLAuthorizationStatus)authorizationStatus {
+#if __IPHONE_OS_VERSION_MAX_ALLOWED >= 140000
+    if (@available(iOS 14.0, *)) {
+        return self.locationManager.authorizationStatus;
+    } else {
+        return kCLAuthorizationStatusNotDetermined;
+    }
+#else
     return [CLLocationManager authorizationStatus];
+#endif
```

Comment:
I'm not sure if this stylistic or a necessity, but do we need the pre processor #if statements?

Reply:
Pragma marks are there to make it compatible with Xcode 11.